.ebextensions/enable_mod_deflate.conf

```diff
@@ -23,10 +23,14 @@ BrowserMatch ^Mozilla/4\.0[678] no-gzip
 
 # MSIE masquerades as Netscape, but it is fine
 BrowserMatch \bMSI[E] !no-gzip !gzip-only-text/html
+</IfModule>
 
 <IfModule mod_headers.c>
 # Make sure proxies don't deliver the wrong content
 Header append Vary User-Agent env=!dont-vary
 </IfModule>
 
-</IfModule>
+# One day for most static assets
+<filesMatch ".(css|jpg|jpeg|png|gif|js|ico)$">
+Header set Cache-Control "max-age=86400, public"
+</filesMatch>
```

Comment:
Not a blocker as it works but is this still needed if the app is setting it?. And why is the expiry of static assets by app and eb different?

Reply:
The application doesn't serve the static assets in Elastic Beanstalk, Apache does. This is the Apache config to enable the caching.